pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -235,9 +242,16 @@ type nodeToUpdateStatusFor struct {
 	volumesToReportAsAttached map[v1.UniqueVolumeName]v1.UniqueVolumeName
 }
 
+func (asw *actualStateOfWorld) MarkVolumeAsUncertain(
+	uniqueName v1.UniqueVolumeName, volumeSpec *volume.Spec, nodeName types.NodeName) error {
+
+	_, err := asw.AddVolumeNode(uniqueName, volumeSpec, nodeName, "", false)
```

Comment:
nit: for unnamed variables used comments to explain i.e.
`asw.AddVolumeNode(uniqueName, volumeSpec, nodeName, "", false /* attached */)`